docs/asciidoc/advanced-contextPropagation.adoc

```diff
@@ -0,0 +1,38 @@
+[[context.propagation]]
+= Context-Propagation Support
+
+Since 3.5.0, Reactor-Core embeds support for the `io.micrometer:context-propagation` SPI.
+This library is intended as a means to easily adapt between various implementations of the concept of a Context, of which
+`ContextView`/`Context` is an example, and between `ThreadLocal` variables as well.
+
+`ReactorContextAccessor` allows the Context-Propagation library to understand Reactor `Context` and `Contextview`.
+It implements the SPI and is loaded via `java.util.ServiceLoader`.
+No user action is required, other than having a dependency on both reactor-core and `io.micrometer:context-propagation`. The `ReactorContextAccessor` class is public but shouldn't generally be accessed by user code.
+
+On top of that, Reactor-Core 3.5.0 also introduces operators that transparently deal with `ContextSnapshot`s if the library is available at runtime, for users' convenience.
+
+== `contextCapture` Operator
+
+This operator is a convenient alternative to `contextWrite` to perform a `ContextSnapshot#capture` and then to use that snapshot to populate the Reactor `Context`.
+
+As a result, if there were any ThreadLocal values during subscription phase, for which there is a registered `ThreadLocalAccessor`, their values would now be stored in the Reactor `Context` and visible
+at runtime in upstream operators.
+
+====
+[source,java]
+----
+//assuming TL is known to Context-Propagation as key TLKEY.
+static final ThreadLocal<String> TL = new ThreadLocal<>();
+
+//in the main thread, tl is set to "HELLO"
+TL.set("HELLO");
+
+Mono.deferContextual(ctx ->
+  Mono.delay(Duration.ofSeconds(1))
+      //we're now in another thread, tl is not set
+      .map(v -> "delayed " + ctx.getOrDefault(TLKEY, "not found") + ", tl=" + TL.get())
```

Comment:
How about

    "delayed ctx[" + TLKEY + "]=" + ctx.getOrDefault(TLKEY, "not found") + ", TL=" + TL.get()

For me at least it would be easier to understand the result.